build/common.sh

```diff
@@ -159,19 +159,21 @@ function _feedback() {
 
   local timestamp
   timestamp="$(date --iso-8601=seconds --utc)"
-  # Use the provided color code label
-  local color
-  color="${1}"
+
+  local log_level
+  log_level="${LOG_LEVEL:-WARNING}"
+
+  # ${!log_level} will set the color code related to log_level
```

Comment:
Since we no longer use `${!color}` this is a bit more ambiguous so I added a comment